index.html

```diff
@@ -58,10 +53,8 @@ <h1  class="primary-text">Hey There. <br>I'm Feven</h1>
                 <h2 class="supporting-text">I'am a Software Developer</h2>
             </div>
            <div>
-            <p class="description">I can help you build a product,feature or website look through some of my work and experience if you like what you see and have a project you need coded,don't hesitate to contact me.</p>
-           </div>
-           <div class="illustration-header">
-            <img src="images/Illustration-Desktop.png" alt="background illustration" class="illustration-header">
+                <p class="description">I can help you build a product,feature or website look through some of my work and experience if you like what you see and have a project you need coded,don't hesitate to contact me.
+                </p>
            </div>
         </div>
         <div class="social-media">
```

Comment:
- The padding between each social icon is less compared to the Figma template. As per the requirement of matching the Figma template as much as possible, do you mind fixing the spacing issue for social icons please? 😃 

Figma Design
<img width="97" alt="Screenshot 2022-12-10 at 10 34 22 PM" src="https://user-images.githubusercontent.com/24832846/206866624-feb36882-634a-4eec-ac58-ce22d194eb72.png">

Your Design
<img width="89" alt="Screenshot 2022-12-10 at 10 32 47 PM" src="https://user-images.githubusercontent.com/24832846/206866653-55580c50-f253-4c2d-9759-153de8e2202f.png">

Reply:
I have increased the padding between links to match the Figma design.